public/app/plugins/panel/geomap/types.ts

```diff
@@ -43,3 +44,23 @@ export interface GeomapPanelOptions {
   basemap: MapLayerOptions;
   layers: MapLayerOptions[];
 }
+export interface FeatureStyleConfig {
+  shape?: string;
+  fill?: Fill; //eventually be ColorDimensionConfig
+  stroke?: Stroke; //eventually be ColorDimensionConfig
```

Comment:
for anything we save in configs (panel JSON) we want to be super careful about what the real shape is.  Fill/Stroke are classes with all kinds of properties.

Maybe we should start with:
* fillColor?: string;
* strokeWidth?: number;